tests/test_formhandler.py

```diff
@@ -601,6 +601,14 @@ def test_date_comparison(self):
             expected.index = actual.index
             afe(actual, expected, check_like=True)
 
+    def test_edit_id_type(self):
+        target = self.copy_file('sales.xlsx', 'sales-edits.xlsx')
+        tempfiles[target] = target
+        args = {'sales': [1], 'date': [0]}
```

Comment:
Should be valid date, else due to recent fix on inferring types, will make this test fail. Will patch this with

```
args = {'sales': [1], 'date': ['2018-01-10']}
```

Reply:
OK. Thanks @pratapvardhan -- please send a PR before %1.51